src/copy.c

```diff
@@ -111,7 +111,7 @@ timescaledb_CopyFrom(CopyChunkState *ccstate, List *range_table, Hypertable *ht)
 	TupleTableSlot *myslot;
 	MemoryContext oldcontext = CurrentMemoryContext;
 
-	ErrorContextCallback errcallback;
+	ErrorContextCallback errcallback = { 0 };
```

Comment:
I would argue that this is not sufficient. If `error_context_stack` is set to some value, but we do not have a copy context (for any reason) it would result in `error_context_stack` being set to NULL after the execution even though it should not be touched.